docs/quickstart.rst

```diff
@@ -35,15 +35,17 @@ one field: ``hello`` and an input name. And when we query it, it should return `
 
 .. code:: python
 
-    import graphene
+    from graphene import ObjectType, Schema, String
 
-    class Query(graphene.ObjectType):
-        hello = graphene.String(argument=graphene.String(default_value="stranger"))
+
+    class Query(ObjectType):
+        hello = String(argument=String(default_value="stranger"))
 
         def resolve_hello(self, info, argument):
-            return 'Hello ' + argument
+            return "Hello " + argument
```

Comment:
f string here?

Reply:
I see you're doing this in https://github.com/graphql-python/graphene/pull/984

Reply:
Absolutely. For this first pass, I (tried!) to ignore actual code outside of the Meta/imports thing, but good catch.

Reply:
Fixed locally. Will push in a bit.